copyright

```diff
@@ -1145,6 +1145,46 @@ Copyright: Arachi-Lover
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (under the same license).
 
+Files:
+ images/effect/firestorm*
+ images/effect/heaver*
+ images/effect/*digger*
+ images/effect/shunt*
+ images/hardpoint/expeller*
+ images/hardpoint/firestorm*
+ images/hardpoint/korath heaver*
+ images/icon/firestorm torpedo*
+ images/icon/korath heaver*
+ images/outfit/expeller*
+ images/outfit/firestorm*
+ images/outfit/korath heaver*
+ images/outfit/*korath reverser*
```

Comment:
while the wildcard is perhaps superfluous for the digger, it is not here.  this would have to be expanded to separate entries for the five sizes of reverser in order to match without the leading wildcard.